src/components/UserInfo.js

```diff
@@ -1,15 +1,38 @@
+import Api from "./Api.js";
+
 export default class UserInfo {
-  constructor(nameElement, titleElement) {
-    this._name = nameElement;
-    this._title = titleElement;
+  constructor({ name, about, avatar, apiHeaders, apiUser }) {
+    this.name = name;
+    this.about = about;
+    this.avatar = avatar;
+    this._headers = apiHeaders;
+    this._endpoint = apiUser;
   }
 
   getUserInfo() {
-    return { name: this._name.textContent, title: this._title.textContent };
+    const userInfoApiGet = new Api({
```

Comment:
Please, don't use api calls inside `UserInfo` because it's only for rendering the user  data on the page